src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -1175,7 +1169,7 @@ module.exports = function<T, P, I, TI, PI, C, CX, PL>(config : HostConfig<T, P,
       // If we're not already inside a batch, we need to flush any task work
       // that was created by the user-provided function.
       if (!isPerformingWork && !isBatchingUpdates) {
-        performWork(TaskPriority);
+        performWork(TaskPriority, null);
```

Comment:
I actually don't know why Flow didn't catch this before, because the type on `performWork` was already `Fiber | null`